### PR TITLE
reps-platform 3.4.12 - eu_myths updated

### PR DESCRIPTION
**Pull request by libetho**

### CODE REVIEW

File: resources/reps-platform.make [hunk 1]
```diff
@@ -5,7 +5,7 @@ core = 7.x
 ; Contributed modules
 ; ===================
 
-projects[admin_language][subdir] = "contrib"
+projects[admin_language][subdir] = "contrib"
```
Comment (zarabatana) [inline resources/reps-platform.make]: Please remove whitespace on end of line.

File: resources/reps-platform.make [hunk 2]
```diff
@@ -31,7 +31,7 @@ projects[media_browser_plus][version] = 3.0-beta4
 projects[media_browser_plus][patch][] = "https://www.drupal.org/files/issues/download-files-in-media-basket-does-not-work-2821063-2.patch"
 
 projects[menu_admin_per_menu][subdir] = "contrib"
-projects[menu_admin_per_menu][version] = 1.0
+projects[menu_admin_per_menu][version] = 1.1
 
 projects[multiform][subdir] = "contrib"
 projects[multiform][version] = 1.4
```
Comment (zarabatana) [inline resources/reps-platform.make:37]: Needs to be updated to minimum version - 1.6

**Review by zarabatana — CHANGES_REQUESTED** [review]

File: resources/reps-platform.make (hunk 2 above)
Comment (libetho) [reply resources/reps-platform.make:37]: @zarabatana 
I don't think there is a version 1.6 of the module Menu Admin per Menu: https://www.drupal.org/project/menu_admin_per_menu/releases?api_version%5B%5D=103

Comment (zarabatana) [reply resources/reps-platform.make:37]: You're right. Last one is 1.1

**Review by zarabatana — APPROVED** [review]

**Pull request merged**